kernel/src/syscall/arch/riscv.rs

```diff
@@ -219,6 +219,8 @@ impl_syscall_nums_and_dispatch_fn! {
     SYS_GETGROUPS = 158          => sys_getgroups(args[..2]);
     SYS_SETGROUPS = 159          => sys_setgroups(args[..2]);
     SYS_NEWUNAME = 160           => sys_uname(args[..1]);
+    SYS_GETRLIMIT = 163          => sys_getrlimit(args[..2]);
+    SYS_SETRLIMIT = 164          => sys_setrlimit(args[..2]);
```

Comment:
Need to update the Linux compatibility page of the Asterinas Book

Reply:
Thanks for the reminding. It is updated now.